circle.yml

```diff
@@ -22,5 +22,6 @@ deployment:
     branch:
       - master
     commands:
+      - ./start_refresh.sh
```

Comment:
Why separated script for just one simple call? We don't do that for e.g. calling runscope webhook.